docs/auth/jwt/tutorials/integrations/4-clerk.mdx

```diff
@@ -32,14 +31,11 @@ Before continuing, ensure you have:
 - A local application that can integrate with Clerk for authentication.
 - A local Hasura DDN project.
```

Comment:
We should add a note for [this issue](https://discord.com/channels/407792526867693568/1313460021278801940/1313569827981627422). By default, Clerk uses the `RSA-OAEP` algorithm, which the DDN engine doesn't support. The user needs to remove the algorithm in `Realm Settings -> Keys -> Add Providers tab`.

Reply:
This is for Realm right? Not Clerk? I think clerk uses [RS256](https://clerk.com/docs/how-clerk-works/tokens-signatures)

Reply:
Nevermind. I mistook Clerk with [Keycloak](https://www.keycloak.org/)